glean_parser/translation_options.py

```diff
@@ -0,0 +1,51 @@
+import pydoc
+
+
+def translate_options(ctx, param, value):
+    text = """Target language options for Translate function
+
+These are backend specific and optional, provide as key:value
+
+Rust:
+- not used
+Swift:
```

Comment:
Let's add a bit of spacing:

```suggestion

Swift:
```